src/controllers/dlgprefcontroller.cpp

```diff
@@ -21,6 +21,17 @@
 #include "preferences/usersettings.h"
 #include "util/version.h"
 
+QString nameForPreset(const PresetInfo& preset);
```

Comment:
Could you also move nameForPreset from below into this anonymous namespace? That way you don't need to add a prototype here.